ChangeLog

```diff
@@ -103,6 +103,9 @@ Changes
    * Make it easier to define MBEDTLS_PARAM_FAILED as assert (which config.h
      suggests). #2671
    * Make `make clean` clean all programs always. Fixes #1862.
+   * Add reproducible option to `ssl_client2` and `ssl_server2` to enable test
```

Comment:
```suggestion
   * Add `reproducible` option to `ssl_client2` and `ssl_server2` to enable test
```
Without quotes it looks like there's a new option and it's reproducible, whatever that means.

Reply:
Indeed better with quotes, ok for the rest :-)